src/managers/config.py

```diff
@@ -415,14 +427,15 @@ def zk_jaas_config(self) -> str:
         Returns:
             String of Jaas config for ZooKeeper auth
         """
-        return f"""
-Client {{
-    org.apache.zookeeper.server.auth.DigestLoginModule required
-    username="{self.state.zookeeper.username}"
-    password="{self.state.zookeeper.password}";
-}};
-
+        return inspect.cleandoc(
```

Comment:
**question** what is this for?

Reply:
Aesthetics. Having it out of alignment was frustrating.